### PR TITLE
Chapter1/homework/9

### PR DESCRIPTION
**Pull request by b4nter**

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/ArrowMovement.cs  10
- Src/BootCamp.Chapter/BinaryConverter.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5027887&bid=16568771)

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5027887)

File: Src/BootCamp.Chapter/ArrowMovement.cs [hunk 2]
```diff
@@ -18,7 +18,30 @@ public static class ArrowMovement
         /// <returns>One of the arrow characters. '↥' by default.</returns>
         public static char GetIndicator(char symbol)
         {
-            return '-';
+            switch (symbol)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/ArrowMovement.cs]: Codacy Issue found: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5027887)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/ArrowMovement.cs  10
- Src/BootCamp.Chapter/BinaryConverter.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5027887&bid=16568771)

File: Src/BootCamp.Chapter/ArrowMovement.cs [hunk 3]
```diff
@@ -18,7 +18,31 @@ public static class ArrowMovement
...
+                    return '↧';
+                case 's':
+                    return '↧';
+                case 'D':
+                    return '↦';
+                case 'd':
+                    return '↦';
+                default:
+                    return '-';
+            }
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/ArrowMovement.cs]: You can simplify this to:
```csharp
switch (symbol)
{
    case 'W':
    case 'w':
        return '↥';
    // and so on.
}
```

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/ArrowMovement.cs  10
- Src/BootCamp.Chapter/BinaryConverter.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5027887&bid=16568771)

File: Src/BootCamp.Chapter/BinaryConverter.cs [hunk 4]
```diff
@@ -8,12 +8,62 @@ public static class BinaryConverter
...
+            return number;
         }
 
+        public static bool IsBinary(string binary)
+        {
+            for (int i = 0; i < binary.Length; i++)
+            {
+                try
+                {
+                    var number = int.Parse(binary[i].ToString());
```
Comment (Almantask) [inline Src/BootCamp.Chapter/BinaryConverter.cs]: You could use try parse instead. if it fails to parse- return false.

**Pull request closed without merging**